site/content/en/docs/manual/advanced/organization.md

```diff
@@ -5,57 +5,128 @@ weight: 2
 description: 'Using organization in CVAT.'
 ---
 
+**Organization** is a feature for teams of several users
+who work together on projects and share tasks.
+
+Create an **Organization**, invite your team members, and assign
+roles to make the team work better on shared tasks.
+
+See:
+
+- [Personal workspace](#personal-workspace)
+- [Create new organization](#create-new-organization)
+  - [Switching between organizations](#switching-between-organizations)
+- [Organization page](#organization-page)
+  - [Invite members into organization](#invite-members-into-organization)
+  - [Delete organization](#delete-organization)
+
 ## Personal workspace
 
-Your `Personal workspace` will display the tasks and projects you've created.
+The account's default state is activated when no **Organization** is selected.
+
+It means that a personal account is used.
```

Comment:
Not really clear lines. 

Maybe write something like this?

When there is not a selected **Organization**, all the created resources are bonded to the personal account with no ability to share them with others. In corresponding place you can see **Personal workspace** indicator.